setup.py

```diff
@@ -40,23 +40,23 @@
 
 PACKAGE_NAME = 'amazon_kclpy'
 JAR_DIRECTORY = os.path.join(PACKAGE_NAME, 'jars')
-PACKAGE_VERSION = '2.0.4'
+PACKAGE_VERSION = '2.0.5'
 PYTHON_REQUIREMENTS = [
     'boto',
     # argparse is part of python2.7 but must be declared for python2.6
     'argparse',
 ]
 REMOTE_MAVEN_PACKAGES = [
     # (group id, artifact id, version),
-    ('software.amazon.kinesis', 'amazon-kinesis-client-multilang', '2.3.6'),
-    ('software.amazon.kinesis', 'amazon-kinesis-client', '2.3.6'),
-    ('software.amazon.awssdk', 'kinesis', '2.16.98'),
-    ('software.amazon.awssdk', 'aws-cbor-protocol', '2.16.98'),
+    ('software.amazon.kinesis', 'amazon-kinesis-client-multilang', '2.3.7'),
+    ('software.amazon.kinesis', 'amazon-kinesis-client', '2.3.7'),
+    ('software.amazon.awssdk', 'kinesis', '2.17.52'),
```

Comment:
Minor: would be nice if we can group all the `awssdk` dependencies together so it's more obvious for future version updates and dependency changes